libs/platform/src/lib/components/search-field/search-field.component.scss

```diff
@@ -45,3 +45,14 @@
         max-width: none;
     }
 }
+
+// text should not be on screen, but also can't be made hidden or display none.
+// for screen reader requirement
+.fd-search-suggestions-voiceover-text {
+    position: absolute;
+    left: -625rem;
+    top: auto;
+    width: 0.0625rem;
+    height: 0.0625rem;
+    overflow: hidden;
```

Comment:
Please reference the source of this block of code: https://webaim.org/techniques/css/invisiblecontent/

Reply:
I am pretty sure this will be needed in future again and is not related to any component specifically, so I suggest you talk to Manju and spend some extra time creating a separate component in css that is not exposed in the documentation. You then add this styling there and in ngx we can have a custom directive or component that will handle all the cases where we need an html element for a11y purposes.

Reply:
raised https://github.com/SAP/fundamental-ngx/issues/5710 to track this.
will modify this, once directive is created.

reference provided.

Reply:
I am sorry @DeepakSap14 , this is no longer allowed in Fundamental-ngx. We can't approve anything that says "will be refactored once something else is done". Mark the PR as blocked and the review will continue once the issue 5710 is merged.

Reply:
@InnaAtanasova using liveAnnouncer from cdk to read the search result. please review.
will raise a PR to remove `off-screen` component from fundamental-styles.

Reply:
You are missing approval from Platform team.